populus/config/base.py

```diff
@@ -53,6 +53,13 @@ def get_master_config(self):
         else:
             return self.parent.get_master_config()
 
+    def unref(self):
+        while [item for item in self.items(flatten=True) if item[0].rpartition(".")[2]=="$ref"]: # NOQA
```

Comment:
Can you move this into a small function `has_references(config)`.  Took me long enough to understand what this line did that I know it won't be clear down the road.

Also, adding a docstring for this method would be good as well.